packages/expect/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jest-matchers",
+  "name": "expect",
```

Comment:
Yep, this is happening @ljharb and @mjackson agreed on this almost a year back. It took us a long time to get everything in shape, and make `jest-matchers` work in browsers etc. @skovhus built an extensive codemod that gets you 90% of the way there: https://github.com/skovhus/jest-codemods/issues/62

Your assumption about more people using `expect` than `jest-matchers` is not correct according to this chart: http://www.npmtrends.com/expect-vs-jest-matchers although I'm willing to believe many people use a single-file bundle of it in a web site; which also means they don't rely on the npm package being called `expect` and are not updating it through npm.

Either way, we'll publish `expect` with version 21. If people want to keep using the old expect, patches can be released to the `1.x` version of the project.